network/dns-cloud/how-to/register-internal-domain.mdx

```diff
@@ -0,0 +1,52 @@
+---
+meta:
+  title: How to register an internal domain name
+  description: This page explains how to register an internal domain name with Scaleway Domains & DNS.
+content:
+  h1: How to register an internal domain name
+  paragraph: This page explains how to register an internal domain name with Scaleway Domains & DNS.
+dates:
+  validation: 2022-10-12
+  posted: 2022-10-12
+---
+
+Domains & DNS is a managed service, that allows you to register a domain name and configure its DNS zones from a graphical interface. It provides support for queries via both IPv4 and IPv6 and supports all common types of DNS records.
+
+Scaleway Domains & DNS is available to manage DNS zones for internal and external domains (domains created at external registrars). 
+
+<Message type="requirement">
+
+- You have an account and are logged into the [Scaleway console](https://console.scaleway.com) 
```

Comment:
to register a domain the fee is billed immediately and the user needs to have entered credit card information in their account